preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -566,18 +564,18 @@ def format_frameshift(result):
     frame_shift_strings = []
     for frame_shift in frame_shifts:
         nuc_abs_list = [
-            f"{nuc["begin"] + 1}-{nuc["end"]}"
+            nuc["begin"] + 1 + "-" + nuc["end"]
             if (nuc["end"] + 1) > nuc["begin"]
-            else f"{nuc["begin"] + 1}"
+            else nuc["begin"] + 1
```

Comment:
do we need to convert this to a string now? Both nuc["begin"] and 1 are ints - before the f-string converted this to a string

Reply:
Ah yes I just checked on the preview and the frameshifts are missing from sequence view - most likely this function raises a warning or otherwise fails: 
<img width="400" alt="image" src="https://github.com/user-attachments/assets/7c212ef5-157b-4a19-ac5b-df11c56ac366">
vs 
<img width="457" alt="image" src="https://github.com/user-attachments/assets/92c7cb81-6db0-4055-8d7a-30de29686be5">
on main

Reply:
Good catch - I wonder if mypy would catch this, I'll check

Reply:
I've refactored this a little - now should be safely returning strings and accepting `int|str` for positions (depending on what Nextclade outputs)

Reply:
oooh! awesome thanks for the tests! I will go over them quickly now to check they make sense to me :-)

Reply:
They look good!